src/test/acceptance/features/confirmation-code-navigation.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Confirmation code navigation
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  Once I have entered my confirmation code
+  If I change the contact details used to receive that code
+  I have to go through the send and enter code part of the application to receive a new code.
+
+  Background:
+    Given I am on the first page of the application
+
+  Scenario: Changing my phone number after using it to receive a code requires me to be sent a new code
+    When I complete the application with valid details, selecting to receive my confirmation code via text
+    And I choose to change my answer to phone number
+    And I enter in a new phone number
+    And I must continue the application from the email address page, including entering a new code
```

Comment:
This line is confusing, what does it have to do with the `email address page` when it's email that's changing?

Reply:
Not sure I understand the comment. We're changing phone number here not email, and the email address page is where you will continue from once you change your phone number.

Reply:
Sorry, my original comment was wrong, I should have said:

This line is confusing, what does it have to do with the email address page when it's **phone number** that's changing?

It's still confusing.

Reply:
Maybe we could re-word this as `And I must continue the application from this point on, including entering a new code`

Reply:
Just realised that we need two version of this, so my suggestion probably isn't valid. It would have to be something like `Then I must complete all steps after the phone number page, including entering a new code`

Reply:
Something certainly needs to change, its quite confusing at it stands, I don't see what the email address page has to do with things when this scenario is updating the phone number and resending the code via text.

Reply:
Updated to `Then I must complete all steps after the phone number page, including entering a new code`